packages/mds-db/geographies.ts

```diff
@@ -65,6 +61,46 @@ export async function readGeographies(params?: {
     throw err
   }
 }
+
+export async function readGeographiesSummary(params?: {
+  get_read_only?: boolean
+}): Promise<GeographySummary[]> {
+  // use params to filter
+  // query on ids
+  // return geographies
```

Comment:
that's a misleading comment

Reply:
I kind of hate code comments in general (especially on a method as trivial as this), so more than happy to remove :)